packages/@react-spectrum/list/src/ListViewItem.tsx

```diff
@@ -222,6 +225,9 @@ export function ListViewItem<T>(props: ListViewItemProps<T>) {
               }
             </div>
           }
+          {isDropTarget && !dropIndicator?.dropIndicatorProps['aria-hidden'] &&
+            <div role="button" {...visuallyHiddenProps} {...dropIndicator?.dropIndicatorProps} ref={dropIndicatorRef} />
+          }
```

Comment:
For other reviewers, this bit is needed to announce the droppable folder/item as a drop indicator. @reidbarber I assume this was added as visually hidden button for a reason? Can we not spread the drop indicator props onto the row itself?

Reply:
I think that works, but this is better. Re: https://github.com/adobe/react-spectrum/pull/3055#discussion_r870484248